### PR TITLE
Fix admin routes and subject + fix cs

### PR DESCRIPTION
**Pull request by qsomazzi**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,4 +8,5 @@ coverage
 composer.lock
 vendor
 node_modules
-.sass-cache
\ No newline at end of file
+.sass-cache
+.php_cs.cache
```
Comment (OskarStark) [inline .gitignore]: is this correct in this PR?

File: Makefile [hunk 2]
```diff
@@ -1,8 +1,8 @@
 cs:
```
Comment (OskarStark) [inline Makefile]: same as ```.gitignore``` file

File: .gitignore (hunk 1 above)
Comment (qsomazzi) [inline .gitignore]: Yes, when you run cs-fixer, you have a cache file that need to be ignored, like here : https://github.com/sonata-project/SonataAdminBundle/blob/master/.gitignore#L10

File: Makefile (hunk 2 above)
Comment (qsomazzi) [inline Makefile]: same, like here : https://github.com/sonata-project/SonataAdminBundle/blob/master/Makefile#L2 :)

**Comment by OskarStark** [comment]
as we don't have a stable version and this is a new bundle, we don't have to stay BC right now, am I right?

File: .gitignore (hunk 1 above)
Comment (OskarStark) [inline .gitignore]: i know, but this has nothing to do with ```Fix admin routes and SetSubject``` so this should be a separate PR IMO

**Comment by qsomazzi** [comment]
Yes, this bundle is in WIP so we don't have to stay BC

File: .gitignore (hunk 1 above)
Comment (qsomazzi) [inline .gitignore]: Will rename my PR and commit to be more clear so

**Comment by qsomazzi** [comment]
@OskarStark if it's ok for you, i will merge this PR since the bundle is in WIP. If you got important feedback on something i did wrong, i don't merge it of course.

**Comment by OskarStark** [comment]
the PR looks ok to me but IMO there a 2 topics in:

- the Code for the PR itself
- some dev changes like php_cs 

i'd like if you would like top remove the Makefile and php_cs stuff from this PR and provide a separate PR

**Comment by qsomazzi** [comment]
ok will do this then

**Comment by OskarStark** [comment]
thank you!
